lib/src/movie_widgets/image_type.dart

```diff
@@ -0,0 +1,4 @@
+enum ImageType {
+  Error,
+  Default,
```

Comment:
Remember, lower camel case on names

Reply:
I can't write default in lower case because it's a reserved word, maybe loading it's fine?

Reply:
You can change default with initial or loading too

Reply:
Done